server/core_authenticate.go

```diff
@@ -431,8 +432,8 @@ func AuthenticateFacebook(ctx context.Context, logger *zap.Logger, db *sql.DB, c
 
 	// Create a new account.
 	userID := uuid.Must(uuid.NewV4()).String()
-	query = "INSERT INTO users (id, username, facebook_id, create_time, update_time) VALUES ($1, $2, $3, now(), now())"
-	result, err := db.ExecContext(ctx, query, userID, username, facebookProfile.ID)
+	query = "INSERT INTO users (id, username, display_name, email, facebook_id, create_time, update_time) VALUES ($1, $2, $3, $4, now(), now())"
```

Comment:
There aren't enough placeholder parameters to match the expected inputs in this SQL.